Dockerfile

```diff
@@ -176,6 +176,7 @@ RUN ./pkg/rpm/build.sh
 COPY cmd/ops_agent_uap_plugin cmd/ops_agent_uap_plugin
 COPY ./builds/ops_agent_plugin.sh .
 RUN ./ops_agent_plugin.sh /work/cache/
+RUN source VERSION && echo $PKG_VERSION  > /work/cache/OPS_AGENT_VERSION
```

Comment:
Should we put this in `./ops_agent_plugin.sh`? This way we don't have to repeat it for each image, and also there's fewer Docker build stages, so fewer layers and faster build time.

Reply:
Done